diff_cover/violations_reporter.py

```diff
@@ -339,6 +344,17 @@ def _run_command(self, src_path):
 
         return stdout.strip().decode(self.STDOUT_ENCODING, 'replace')
 
+    def _run_command_simple(self, command):
```

Comment:
Yeah. I deferred to coverage.py on this, and added tests. Made sense to be explicit about the method anyway.